draft-btw-add-home.txt

```diff
@@ -993,11 +993,11 @@ Internet-Draft       Encrypted DNS in Home Networks         January 2021
    The client will use DEER [I-D.pauly-add-deer] to discover the DoH
    templates supported by the DNS server at the Locator address.  In
    addition to the checks included in DEER, clients should verify the
-   ADN matches the hostname in the DEER-provided templates.  The IP
-   address of the DEER-discovered resolver may differ from the Locator
-   field value so long as the domain name matches the ADN.  This will
-   allow ISPs to have local forwarders send clients to off-network but
-   ISP-owned resolvers whose IP address may not be known in advance.
+   ADN is valid for the certificate provided by the DoH resolver.  The
+   IP address of the DEER-discovered resolver may differ from the
+   Locator field value so long as the domain name matches the ADN.  This
```

Comment:
Same comment as before here - suggest something like "the domain name of the certificate matches..." or similar

Reply:
Ay, I can't believe I missed that. That'll teach me to do rapid feedback iteration. 

I've simply removed the duplication of the ADN requirement to avoid future edit complications. Thanks.